Sdtrig.tex

```diff
@@ -319,6 +319,22 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+For virtual address matches without a mask, \RcsrTdataTwo must be able to hold
+all valid virtual addresses but it need not be capable of holding other values.
+Implementations may convert invalid virtual addresses to a different invalid
+virtual address before comparing the address to \RcsrTdataTwo.
+
+\begin{commentary}
+    A straightforward trigger implementation would compare the effective address
+    to the contents of \RcsrTdataTwo. This is most intuitive to a user setting
+    triggers. However, for instruction fetches the effective address might not
+    be preserved in the hart's state so it is impossible to compare against the
+    effective address. To simplify the spec and allow for more implementations,
```

Comment:
I would say something more like "The PC might not be capable of holding all invalid addresses and some invalid addresses may map to other invalid addresses, making it impossible to compare against the original value that software attempted to write to the PC."  (I intentionally tried to avoid saying "jump" because you could jump there, you could mret/sret there, you could be right near the edge and branch there, you could sequentially execute off the end of the address space, you could be in Sv39 mode at a legal negative address and then write satp to put yourself in Bare mode where it's suddenly illegal to have ones in the top PC bits, etc.)

Reply:
Done.